client/my-sites/checkout/composite-checkout/test/use-cached-domain-contact-details.tsx

```diff
@@ -59,12 +59,11 @@ function MyTestWrapper( {
 }
 
 function MyTestContent( { countries }: { countries: CountryListItem[] } ) {
-	useWpcomStore();
 	const { responseCart, reloadFromServer, updateLocation } = useShoppingCart(
 		initialCart.cart_key
 	);
 	useCachedDomainContactDetails( () => null, countries );
-	const contactInfo = useSelect( ( select ) => select( 'wpcom-checkout' ).getContactInfo() );
+	const contactInfo = useSelect( ( select ) => select( CHECKOUT_STORE ).getContactInfo(), [] );
```

Comment:
It's interesting that TS missed that! Likely because it's in a test and nothing specifically imports it as a module.

Reply:
Yeah, TS does not scan the tests because they're not imported. I often find TS errors in them when I go to do something else.